lib/plugins/aws/invokeLocal/index.test.js

```diff
@@ -352,12 +360,26 @@ describe('AwsInvokeLocal', () => {
           {},
           undefined
         )).to.be.equal(true);
-        delete awsInvokeLocal.options.functionObj.runtime;
       });
     });
 
-    it('throw error when using runtime other than Node.js or Python', () => {
+    it('should call invokeLocalJava when java8 runtime is set', () => {
       awsInvokeLocal.options.functionObj.runtime = 'java8';
+      return awsInvokeLocal.invokeLocal()
+        .then(() => {
+          expect(invokeLocalJavaStub.calledOnce).to.be.equal(true);
+          expect(invokeLocalJavaStub.calledWithExactly(
+            'java',
+            'handler.hello',
+            undefined,
+            {},
+            undefined
+          )).to.be.equal(true);
+        });
+    });
+
+    it('throw error when using runtime other than Node.js or Python', () => {
+      awsInvokeLocal.options.functionObj.runtime = 'go';
```

Comment:
Could we change this to `invalid-runtime` or something like that (so that it won't blow up if we add support for Go)?